tests/unit/CMakeLists.txt

```diff
@@ -566,14 +566,16 @@ function(seastar_gen_mtls_certs)
   set(CLIENT1_SUBJECT "/C=GB/ST=London/L=London/O=Redpanda Data/OU=Core/CN=client1.org")
   set(CLIENT2_SUBJECT "/C=GB/ST=London/L=London/O=Redpanda Data/OU=Core/CN=client2.org")
 
-  # Some openssl commands read and write the serial number file (.srl). These can't be executed in parallel.
+  # The "openssl x509" commands have "-CAcreateserial" in their command lines.
+  # They read and write the serial number file (mtls_ca.srl in this case).
+  # These can't be executed in parallel.
   set_property(GLOBAL APPEND PROPERTY JOB_POOLS openssl_serial=1)
 
   add_custom_command(OUTPUT mtls_ca.key
     COMMAND ${OPENSSL} ecparam -name prime256v1 -genkey -noout -out mtls_ca.key
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   )
-  add_custom_command(OUTPUT mtls_ca.crt mtls_ca.srl
+  add_custom_command(OUTPUT mtls_ca.crt
```

Comment:
Maybe this is right and maybe this is wrong, one thing I'm certain is that I don't know how to review it.

How about dropping -CAcreateserial and using -set_serial nnn instead? We can hardcode 1, 2, 3 for the files we create.

Reply:
We can drop the job pool too.

Reply:
indeed, much better!